app/Commands/AwsAssumeRole.php

```diff
@@ -95,6 +91,8 @@ public function handle()
 
         $assumePrompt = "{$sessionUser}:{$accountName}";
 
+        $envVars['AWS_S3_ENV'] = $accounts->firstWhere('id', $accountId)['s3env'];
```

Comment:
While this works fine, I'd suggest we maybe set `$this->account = the array` at the start to avoid multiple `firstWhere`s

Reply:
Done